### DISCUSSION TITLE
Where in the world can I find the TF-06 micro SD socket for centronics version???

### DISCUSSION THREAD

**Discussion by badass-boss** [BlueSCSI/BlueSCSI-v2#185] [Build it Yourself]
In the centronics version of BlueSCSI there is a micro SD socket called "TF-06_MicroSD". What even is this socket and where can I find it on mouser or digikey or anywhere else?? I searched everywhere on the net and found nothing... Really frustrating. What is this even supposed to be? Or is this BOM outdated?

**Comment by erichelgeson** [comment]
You can find the part numbers in the BOM (Bill Of Materials) files for each version. For example the Centronics BOM can be found here - https://github.com/BlueSCSI/BlueSCSI-v2/blob/main/cc-nc-hardware/Centronics_50Pin/ProductionFiles_JLCPCB/Centronics_50Pin_BOM_SetVR.csv#L19

LCSC Part #C693844 - https://www.lcsc.com/product-detail/SD-Card-Memory-Card-Connector_Korean-Hroparts-Elec-TF-06A_C693844.html

I do not have a mouser or digikey link handy, but if you find it on those sites please share here- thanks!
